BioSQL/BioSeqDatabase.py

```diff
@@ -389,6 +413,13 @@ def fetch_dbid_by_dbname(self, dbname):
         return rv[0][0]
 
     def fetch_seqid_by_display_id(self, dbid, name):
+        """Return the internal id for a sequence using its display id.
+
+        Argunments:
+            - dbid - the internal id for the sub-database
+            - name - the name of the sequence. Corresponds to the
+            name column of the bioentry table of the SQL schema
```

Comment:
Please check this is valid RST, I suspect the final line should be two spaces more indented as a continuation of the bullet point.